README.md

```diff
@@ -15,7 +21,7 @@
 
   ```elixir
   def deps do
-    [{:ueberauth_auth0, "~> 0.3"}]
+    [{:ueberauth_auth0, github: "vivakit/ueberauth_auth0", branch: "big-update"}]
```

Comment:
Kinda hard to merge this in with all the changes to the github urls. 😭

Reply:
Yeah. If this PR is for this repo, then it does not make sense to change the Github URL.